lib/src/config_provider/spec_utils.dart

```diff
@@ -351,6 +351,8 @@ String llvmLibExtractor(dynamic value) {
 }
 
 bool llvmLibValidator(List<String> name, dynamic value) {
+  _logger.severe(
```

Comment:
@dcharkes We are emitting a warning here.

Reply:
My bad. Maybe just make it a warning instead of severe.

Reply:
Done.